entries/entry05.md

```diff
@@ -1,8 +1,60 @@
 # Entry 5
-##### X/X/XX
+##### 3/16/20
 
-Text
+### Update
+In the past weeks me and my partner, ZhiYuan, have continued our research on Machine Learning. I personally looked into the mathematical equations that are required in Machine Learning in order to better understand the code as well as familiarizing myself with syntax and method of the new python module mentioned in my previous blog entry. As of now we do have a Minimum Viable Product which is the algorithmic bot. However this does not conclude this project since we aim to make a Artificial Intelligence bot from the very beginning. My partners’s main focus as of now is improving the algorithmic bot. We hope that one day we can match the bot against each other to compare the strength and weakness of AI and algorithm.
+<br><br>
+### Engineering Design Process
+As for the Engineering Design Process currently we are in the **planning** phase. We have decided to create a q-learning bot as mentioned in the previous blog entry but there is much to be done beside declaring where we are going. We touch upon a bit of the **prototype** phrase with the help of online tutorials.
+<br><br>
+### Knowledge
+While going through the online ML Lessons provided by [Andrew Ng](https://online.stanford.edu/instructors/andrew-ng), a professor at Stanford. I came across the topic of Vectors and Matrices which for me at first reminded me of arrays in Java. This familiarity soon came to an end as it is much more complex than that since you can apply mathematical operation to it.
```

Comment:
Yes, these are not easy! So cool that you're uncovering them now, though! If you learn them in college, it'll be great that you will have already seen an application of them!